core/liquidity/engine.go

```diff
@@ -306,6 +261,18 @@ func (e *Engine) ProvisionsPerParty() ProvisionsPerParty {
 	return e.provisions.ProvisionsPerParty
 }
 
+// SaveLPOrders sets LP orders that have been cancelled from the book but we need to know
+// what they were to recalculate and redeploy.
+func (e *Engine) SaveLPOrders() {
+	for _, p := range e.provisions.ProvisionsPerParty {
+		e.lpPartyOrders[p.Party] = e.orderBook.GetOrdersPerParty(p.Party)
```

Comment:
you will have all the order of the LP here, not only those created by the liquidity engine, is that OK?

Reply:
Would you not want to use GetLiquidityOrders(party) here?

Reply:
Yea good spot, changed it.